app/components/Experience/Experience.scss

```
@@ -1,3 +1,17 @@
+@use '~/assets/styles/_placeholders.scss' as *;
```

Stylelint (17.6.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)